### PR TITLE
Move m2lines & LEAP hubs to regional  HA clusters

### PR DESCRIPTION
**Pull request by yuvipanda**
These are less prone to k8s API failures, like those reported in https://github.com/2i2c-org/infrastructure/issues/1248

We now default to provisioning regional HA clusters - these cost ~70$ a month,
but I think the added reliability is worth it

Ref https://github.com/2i2c-org/infrastructure/issues/1102

### CODE REVIEW

**Comment by yuvipanda** [comment]
You can't 'upgrade' from a non-HA cluster to a HA cluster, needs a redeploy. I've redeployed both the LEAP and m2lines clusters :)

File: docs/howto/operate/new-cluster/new-cluster.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ The _minimum_ inputs this file requires are:
...
+- `regional_cluster`: Set to true to provision a [GKE Regional
+  Highly Available cluster](https://cloud.google.com/kubernetes-engine/docs/concepts/regional-clusters).
+  Costs ~70$ a month, but worth it for the added reliability for most
+  cases except when cost saving is an absolute requirement. Defaults
+  to `true`.
+- `zone`: Zone where cluster nodes and filestore for home directory
+  are created.
+- `region`: Region where cluster master (if `regional_cluster` is
+  `true`) is run, as well as any storage buckets created with
+  `user_buckets`.
```
Comment (GeorgianaElena) [inline docs/howto/operate/new-cluster/new-cluster.md:42]: What do you think about adding these extra inputs (`regional_cluster`, `zone`, `region`) to the example below too: 

https://github.com/2i2c-org/infrastructure/blob/1b45e2fe7f1dce300be12996622e3786aeccc054/docs/howto/operate/new-cluster/new-cluster.md?plain=1#L46-L52

Also, I just noticed there are some extra  ` ```` ` there too

**Review by GeorgianaElena — APPROVED** [review]
Thanks @yuvipanda! I've left a small, docs-related comment, but this looks good to me 🚀

File: docs/howto/operate/new-cluster/new-cluster.md (hunk 1 above)
Comment (sgibson91) [reply docs/howto/operate/new-cluster/new-cluster.md:42]: 'the extra `````` matches this line, so it is correct :) https://github.com/2i2c-org/infrastructure/blob/1b45e2fe7f1dce300be12996622e3786aeccc054/docs/howto/operate/new-cluster/new-cluster.md#L26

Comment (GeorgianaElena) [reply docs/howto/operate/new-cluster/new-cluster.md:42]: Ah, right! Thanks @sgibson91 <3

Comment (yuvipanda) [reply docs/howto/operate/new-cluster/new-cluster.md:42]: @GeorgianaElena added these additional variables to the example!

**Comment by yuvipanda** [comment]
Linkcheck failed with failure for canada.cloudbank.2i2c.cloud, which is from #1249. I'll investigate separately!

**Pull request merged**